service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
```

Comment:
I mean, they'd want to run them eventually. What I'm saying is that you should be able to set up the test environment without immediately having the tests run. These are two distinct pieces of functionality. And even if we ignore the advantage of having them separate on an individual's dev machine, they are also intentionally separate for Travis. You can see what the difference between the Travis `install` (which installs dependencies, sets up the environment, etc.) and `script` (which runs the build or any tests) are here: https://docs.travis-ci.com/user/customizing-the-build/. Basically, if any step in the `install` phase fails, the entire Travis run exits on an error, because if any stage in setup fails, it's assumed any subsequent tests or builds in the `script` phase will not have the required dependencies to succeed. If any stage in the `script` phase fails, however, the remainder will still run. This allows multiple test suites to be run. We actually already have this now. We have the build portion of `script`, which performs all the building, including linking for gcsutils. We also will have the ginkgo tests. It doesn't make sense that if the building stage fails, the ginkgo tests shouldn't be allowed to run. We should have the build and ginkgo portions of `script` on two separate lines under `script`, and have the setup in the `install` step so that we know that when the `script` phase is reached, the environment has been set up correctly.

`install` isn't so much an advanced Travis option as a standard used across the board in repos which rely on Travis. I hope this makes sense. I'd be happy to talk to you more offline about it!